### PR TITLE
Add missing documentation

### PR DESCRIPTION
**Pull request by FrederikSchlemmer**

### CODE REVIEW

File: src/main/java/de/adesso/gitstalker/core/Tasks/RequestProcessorTask.java [hunk 1]
```diff
@@ -31,6 +31,7 @@
     /**
      * Scheduled task checking for queries without crawled information.
      * After picking one query the request starts with the specified information out of the selected query. After the request the query is saved in the repository with the additional response data.
+     * The request can generate a exception because of no remaining Rate Limit.
```
Comment (dariobraun) [inline src/main/java/de/adesso/gitstalker/core/Tasks/RequestProcessorTask.java]: "...**an** exception if the Rate Limit is exhausted."

File: src/main/java/de/adesso/gitstalker/core/Tasks/RequestProcessorTask.java [hunk 2]
```diff
@@ -45,22 +46,18 @@ private void crawlQueryData() {
...
-        for (Query createdQuery : processingQueries) {
-            if (RateLimitConfig.getRemainingRateLimit() - createdQuery.getEstimatedQueryCost() >= 0) {
-                return createdQuery;
-            }
-        }
-        return null;
-    }
-
+    /**
+     * Check whether prioritized requests have already been completed. Favors the application flow so that paused requests are not processed.
```
Comment (dariobraun) [inline src/main/java/de/adesso/gitstalker/core/Tasks/RequestProcessorTask.java]: "~Favors~ **Prioritizes** application flow...."

File: src/main/java/de/adesso/gitstalker/core/processors/ExternalRepoProcessor.java [hunk 3]
```diff
@@ -46,6 +61,11 @@ public void processResponse(Query requestQuery, RequestRepository requestReposit
         super.doFinishingQueryProcedure(this.requestRepository, this.organizationRepository, organization, requestQuery, RequestType.EXTERNAL_REPO);
     }
 
+    /**
+     * When all requests have been processed, a link is created from other collected information. This is done by linking the external repositories to the contributors.
```
Comment (dariobraun) [inline src/main/java/de/adesso/gitstalker/core/processors/ExternalRepoProcessor.java]: "... created ~from~ **to** other collected..."

File: src/main/java/de/adesso/gitstalker/core/processors/OrganizationValidationProcessor.java [hunk 4]
```diff
@@ -36,12 +48,21 @@ public void processResponse(Query requestQuery, RequestRepository requestReposit
         }
     }
 
+    /**
+     * Generates a organization wrapper object if the organization isn't assigned already.
```
Comment (dariobraun) [inline src/main/java/de/adesso/gitstalker/core/processors/OrganizationValidationProcessor.java]: "Generates **an** organization wrapper..."

**Comment by FrederikSchlemmer** [comment]
@dariobraun should be good to go!

**Pull request merged**